embedded/sql/stmt.go

```diff
@@ -3999,3 +4006,59 @@ func (stmt *FnDataSourceStmt) resolveListIndexes(ctx context.Context, tx *SQLTx,
 
 	return newValuesRowReader(tx, params, cols, stmt.Alias(), values)
 }
+
+func NewDeleteTableStmt(table string) *DeleteTableStmt {
+	return &DeleteTableStmt{table: table}
+}
+
+// DeleteTableStmt represents a statement to delete a table.
+type DeleteTableStmt struct {
```

Comment:
DropTableStmt might be more aligned with sql concepts

Reply:
ack, done